### PR TITLE
remove message to manually upgrade data-plane

### PR DESCRIPTION
**Pull request by rgildein**
Since the dev/data-plane is introducing upgrade for data-plane there is no reason to left this message here.

fixes: #140

### CODE REVIEW

File: cou/cli.py [hunk 1]
```diff
@@ -137,9 +137,8 @@ async def get_upgrade_plan(args: CLIargs) -> None:
     :param args: CLI arguments
     :type args: CLIargs
     """
-    analysis_result, upgrade_plan = await analyze_and_plan(args)
+    _, upgrade_plan = await analyze_and_plan(args)
```
Comment (agileshaw) [inline cou/cli.py:140]: I don't think there's any other place needs the analysis_result output from `analyze_and_plan` function. We can change that function to only return `upgrade_plan` and you don't need to ignore an value here.

**Review by agileshaw — CHANGES_REQUESTED** [review]

File: cou/cli.py (hunk 1 above)
Comment (rgildein) [reply cou/cli.py:140]: True, I do not think that this is really related with desire goal of this fix, but I removed it.

**Review by aieri — APPROVED** [review]

**Review by agileshaw — CHANGES_REQUESTED** [review]
Could you also update the docstring of `analyze_and_plan` function?

**Comment by rgildein** [comment]
> `analyze_and_plan`

I do not like this redundancy (type hinting in docstring and real one).

**Review by gabrielcocenza — APPROVED** [review]

**Pull request merged**